src/run.php

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env php
+<?php
+// src/run.php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+use Consolidation\AnnotatedCommand\AnnotatedCommandFactory;
+use Consolidation\AnnotatedCommand\CommandFileDiscovery;
+
+$setAutoloader();
+$terminus = new Terminus();
+$command_factory = new AnnotatedCommandFactory();
+//$formatter_manager = new FormatterManager();
+
+$discovery = new CommandFileDiscovery();
+$command_files = $discovery->discover(__DIR__, '\Commands');
+foreach ($command_files as $command_class) {
+  $command_instance = new $command_class();
+  //$command_factory->commandProcessor()->setFormatterManager($formatter_manager);
+  $command_list = $command_factory->createCommandsFromClass($command_instance);
+  foreach ($command_list as $command) {
+    $terminus->add($command);
+  }
+}
+
+/**
+  * Sets the autoloader up to retrieve files as necessary
+  *
+  * @return void
+  */
+function setAutoloader() {
```

Comment:
I did it this way because the old and the new Terminus are still using the same `composer.json`. It'll make sense to move it into `composer.json` once we're ready to remove the `php` dir.